app/components/Conflict.js

```diff
@@ -217,6 +228,9 @@ export default class Conflict extends React.Component {
     if (updatedFeature.isDeleted) {
       return this.upstreamDeleted()
     }
+    if (updatedFeature.geometry.type === 'LineString' || updatedFeature.geometry.type === 'Polygon') {
```

Comment:
I think we should check if this is a complex edit and then disallow overwriting. The current condition will stop the user from overwriting attribute changes.

Reply:
I'm not sure actually. Perhaps this is ok for now.